addons/stock/tests/test_quant.py

```diff
@@ -47,7 +47,15 @@ def setUpClass(cls):
             'type': 'product',
             'tracking': 'serial',
         })
-        cls.stock_location = cls.env.ref('stock.stock_location_stock')
+        cls.stock_location = cls.env['stock.location'].create({
+            'name': 'stock_location',
+            'usage': 'internal',
+        })
+        cls.stock_subloc2 = cls.env['stock.location'].create({
```

Comment:
i don't see where stock_subloc2 is used?

Reply:
@sle-odoo this [testcase](https://github.com/odoo/odoo/pull/34258/files/b5974fdfb23ebe17ffc17ac67799e219ff4c4752#diff-eef6aab50d6b1f825856390112d96c1dR227) is using the child location of the stock_location. so, that's way we have to create it.